CHANGELOG.md

```diff
@@ -43,13 +43,15 @@ collected logs to standard output. This can be useful for debugging. (@jcreixell
   - `otelcol.processor.memory_limiter` periodically checks memory usage and
     drops data or forces a garbage collection if the defined limits are
     exceeded. (@tpaschalis)
-  
+
 ### Enhancements
 
 - Update OpenTelemetry Collector dependency to v0.61.0. (@rfratto)
 
 - Update Prometheus dependency to v2.38.0. (@rfratto)
 
+- Add caching to Prometheus relabel component. (@mattdurham)
```

Comment:
* Can you prefix this indicating that it's a Flow-specific change 
* Can you change this to explain the effect to end users rather than what was done? (for example: reduce CPU usage of relabeling)

Reply:
(also, mentioning the specific component name `prometheus.relabel` here would be nice)